docker/base/lifecycle/post-start.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+AUTOSAVE_BRANCH_PREFIX="auto-save-$JUPYTERHUB_USER"
```

Comment:
It would be nice to group the branches with renku prefix since there might isolation branches too:
```suggestion
AUTOSAVE_BRANCH_PREFIX="renku/autosave/${SERVER_NAME}"
```

Reply:
why `SERVER_NAME`? it's a function of `namespace`, `project` and `commit-sha` -- we need to be able to identify the user

Reply:
namespace and project are already part of the remote where you are pushing the branch

Reply:
I know -- what I'm asking is why you want to include `SERVER_NAME`

Reply:
if you relaunch the server from the same sha1 will the server name be the same?

Reply:
if you relaunch the server from the same sha1 will the server name be the same?

Reply:
yes - but I would prefer to keep the sha and original branch name as part of the autosave branch ref to be easier to identify

Reply:
fine, just use / as separator for nice branch grouping to dirs

Reply:
👍 yes I like that

I think we should keep these names as human-parseable as possible since users might need to identify them at some point

Reply:
so I think we can go with 

`AUTOSAVE_BRANCH_PREFIX="renku/autosave/${JUPYTERHUB_USER}"`